src/clientiface.cpp

```diff
@@ -252,7 +252,8 @@ void RemoteClient::GetNextBlocks (
 			*/
 
 			float camera_fov = (72.0*M_PI/180) * 4./3.;
-			if(isBlockInSight(p, camera_pos, camera_dir, camera_fov, 10000*BS) == false)
+			if(isBlockInSight(p, camera_pos, camera_dir, camera_fov,
+					full_d_max * BS * (MAP_BLOCKSIZE + 1)) == false)
```

Comment:
This exactly fixes so that the radius of the sphere is the same as the distance to the cube sides. Possible that full_d_max + 1 ... does that too. Perhaps simply `+ BS` does it too.